src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -99,9 +100,30 @@ private static bool Contract_Update(ApplicationEngine engine)
 
             private static bool Contract_Destroy(ApplicationEngine engine)
             {
+                if (!engine.TryPop(out StackItem sendTo)) return false;
+
                 UInt160 hash = engine.CurrentScriptHash;
                 ContractState contract = engine.Snapshot.Contracts.TryGet(hash);
                 if (contract == null) return true;
+
+                // Send funds
+
+                UInt160 sendToHash = new UInt160(sendTo.GetSpan());
+                if (engine.Snapshot.Contracts.TryGet(sendToHash)?.Payable == false) return false;
+
+                var balance = NativeContract.NEO.BalanceOf(engine.Snapshot, sendToHash);
+                if (balance > 0)
```

Comment:
Why check the balance of sendToHash? Does this mean that the amount of payee's account is 0 and no money can be received?